.circleci/config.yml

```diff
@@ -158,7 +158,7 @@ jobs:
       - run:
           name: Get Roxctl binary and set the image name
           command: |
-            cci-export IMAGE_NAME "quay.io/rhacs-eng/main:3.0.55.x-4-gd2e48c0fd6"
+            cci-export IMAGE_NAME "quay.io/rhacs-eng/main:3.70.0"
```

Comment:
How about using latest build and run this job nightly to detect similar issues? This can be done in a separate PR

Reply:
Discussed this offline with @janisz and agreed on merging and releasing this PR for now, addressing the nightly run within this ticket: [ROX-11371](https://issues.redhat.com/browse/ROX-11371).